src/transport.ts

```diff
@@ -90,7 +90,7 @@ export class WebRTCTransport implements Transport {
     // create offer sdp
     let offerSdp = await peerConnection.createOffer();
     // generate random string for ufrag
-    const ufrag = genUuid().replaceAll('-', '');
+    const ufrag = "libp2p+webrtc+v1/" + genUuid().replaceAll('-', '');
```

Comment:
why is this prefix added?
The comment mentions what the code is doing, but not its intention, which is pretty much the only thing a comment can be useful for (imho)

Reply:
This is part of the spec: https://github.com/libp2p/specs/pull/412#discussion_r1007462445. I will add a comment as you suggested.